.github/workflows/stale.yml

```diff
@@ -0,0 +1,24 @@
+name: Mark stale issues and pull requests
+
+on:
+  schedule:
+  - cron: '30 * * * *'
+
+jobs:
+  stale:
+
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+
+    steps:
+    - uses: actions/stale@v3
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Should this be set up as an org secret instead of repo secret?

Reply:
It's not a token we actually set. Any token that starts with `GITHUB_` is reserved by GitHub for their use.

See https://docs.github.com/en/actions/reference/authentication-in-a-workflow

I had the same confusion and tried to set a PAT for Pralor with that value. GitHub won't let you.

Reply:
AH, right ok that makes sense!